### PR TITLE
:sparkles: Add deleteChipGroup handler

### PR DESCRIPTION
**Pull request by rszwajko**
Resolves: #1744

### CODE REVIEW

**Comment by rszwajko** [comment]
### Before
Screenshot from 2024-03-07 18-17-00

### After
Screenshot from 2024-03-07 18-17-12

**Comment by ibolton336** [comment]
https://github.com/konveyor/tackle2-ui/issues/1744

**Review by ibolton336 — APPROVED** [review]
LGTM!

**Review by aufi — APPROVED** [review]

File: client/src/app/components/FilterToolbar/MultiselectFilterControl.tsx [hunk 1]
```diff
@@ -357,6 +357,7 @@ export const MultiselectFilterControl = <TItem,>({
       id={`filter-control-${category.key}`}
       chips={chips}
       deleteChip={(_, chip) => onFilterClear(chip)}
+      deleteChipGroup={() => setFilterValue([])}
```
Comment (sjd78) [inline client/src/app/components/FilterToolbar/MultiselectFilterControl.tsx:360]: Passing thought -- to be more consistent with the deleteChip prop, add a `onFilterClearAll()` function and put it right next to `onFilterClear` so the actual work of clearing the filter values are in the same place.

**Pull request merged**